decidim-verifications/app/jobs/decidim/verifications/csv_census/process_census_data_job.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Verifications
+    module CsvCensus
+      class ProcessCensusDataJob < ApplicationJob
+        queue_as :default
+
+        def perform(data, organization, user)
+          imported_data = []
+          failed = []
```

Comment:
```suggestion
          @imported_data = []
          @failed = []
```
If you use instance variables, you can add this information in the entry log (or counters)

Reply:
Done